aws-lambda/src/databricks_cdk/resources/permissions/changes.py

```diff
@@ -0,0 +1,87 @@
+from databricks.sdk.service.catalog import PermissionsChange, PermissionsList, Privilege, PrivilegeAssignment
+
+
+def get_assignment_dict_from_permissions_list(permissions_list: PermissionsList) -> dict[str, list[Privilege]]:
+    """Converts PermissionsList to dict with key of principal and list of associated privileges as value"""
+    privilige_assignments = permissions_list.privilege_assignments or {}
+
+    if privilige_assignments is None:
```

Comment:
This can never be None, because of the or operator in line 6. I think you want to check for empy dict here instead.

Reply:
Yes you are correct. Removed the or statement on line 6